gke/cluster.tf

```diff
@@ -21,10 +21,11 @@ resource "google_container_node_pool" "converge_node_pool" {
 
   management {
     auto_upgrade = true
+    auto_repair  = true
   }
 
   node_config {
-    machine_type = "g1-small"
+    machine_type = "n1-standard-1"
```

Comment:
I recommend storing these values as environment variables, so you won't have to raise another PR in order to change something in the future.

Reply:
Our staging is down currently and we don’t intend to change the machine type, at least not anytime soon.